contrib/init/bitcoind.openrc

```diff
@@ -89,4 +91,15 @@ checkconfig()
 		eerror ""
 		return 1
 	fi
+
+	if ! grep -qs '^alertnotify=' "${BITCOIND_CONFIGFILE}" ; then
+		ewarn ""
+		ewarn "It is recommended that you set alertnotify in your bitcoin.conf"
+		ewarn "so you are notified of problems:"
+		ewarn ""
+		ewarn "ie: alertnotify=echo %%s | mail -s \"Bitcoin Alert\"" \
+			"admin@foo.com"
+		ewarn ""
+		return 0
+	fi
```

Comment:
This seems excessive.

Reply:
I wanted to keep it since the existing script was also notifying about it. Do you think this part should be removed?

Reply:
The existing notice is when misconfigured, not every time you start it.